phabricatoremails/constants.py

```diff
@@ -3,6 +3,7 @@
 #  file, You can obtain one at http://mozilla.org/MPL/2.0/.
 
 # Strings used to define statistics
-STAT_FAILED_TO_SEND_MAIL = "mail.outgoing.sent"
+STAT_PROCESSED_MAIL = "mail.processed.sent"
```

Comment:
I assume that the old/abandoned stats under `mail.outgoing.sent` will be defunct and that's OK.

Reply:
I don't believe they're in use yet, yeah.